### PR TITLE
Add more info about inbound filters

### PR DESCRIPTION
**Pull request by roggenkemper**
this pr adds more information about the web crawler and browser extension inbound filters

### CODE REVIEW

File: docs/product/data-management-settings/filtering/index.mdx [hunk 1]
```diff
@@ -33,6 +33,14 @@ Legacy browser filters were updated in Feb 2024 and will be periodically evaluat
 
 If you had a legacy browser filter on before the update, the old filter will appear in your settings as "Deprecated". Deprecated legacy browser filters still work. However, if you turn them off, you won’t be able to turn them on again and will need to use the new filters instead.
 
+### Browser Extension Errors
+
+The browser extension filter checks an event to see if it matches any known browser extension errors. The filter will check both for the error message to see if the error is a known to come from browser extensions and it will check the source of the event, to see if it is coming from a browser extension.
+
+### Web Crawler Errors
+
+The web crawler filter allows you to filter out errors from web crawlers from the following sites: Baidu, Yahoo, Sogou, Facebook, Alexa, generic bots and spiders, Slack, Google Indexing, Pingdom, Lytics, AWS Security Scanner, Hubspot, and Bytedance. Errors from Slack's Slackbot web crawler will not be filtered.
```
Comment (vivianyentran) [inline docs/product/data-management-settings/filtering/index.mdx:42]:
```suggestion
Due to their nature, web crawlers often encounter errors that normal users won't see. You can use the web crawler filter to filter out errors from web crawlers from the following sites: Baidu, Yahoo, Sogou, Facebook, Alexa, Slack, Google Indexing, Pingdom, Lytics, AWS Security Scanner, Hubspot, Bytedance, and other generic bots and spiders. Errors from Slack's Slackbot web crawler will not be filtered. 
```

File: docs/product/data-management-settings/filtering/index.mdx [hunk 2]
```diff
@@ -33,6 +33,14 @@ Legacy browser filters were updated in Feb 2024 and will be periodically evaluat
 
 If you had a legacy browser filter on before the update, the old filter will appear in your settings as "Deprecated". Deprecated legacy browser filters still work. However, if you turn them off, you won’t be able to turn them on again and will need to use the new filters instead.
 
+### Browser Extension Errors
+
+The browser extension filter checks an event to see if it matches any known browser extension errors. The filter will check both for the error message to see if the error is a known to come from browser extensions and it will check the source of the event, to see if it is coming from a browser extension.
```
Comment (vivianyentran) [inline docs/product/data-management-settings/filtering/index.mdx:38]:
```suggestion
Some browser extensions are known to cause errors. You can use the browser extension filter to filter these out. The filter checks the error message and the event source to see if it's a known error coming from a browser extension.
```

**Review by vivianyentran — APPROVED** [review]
added some wording suggestions but otherwise looks good!

File: docs/product/data-management-settings/filtering/index.mdx [hunk 3]
```diff
@@ -33,6 +33,14 @@ Legacy browser filters were updated in Feb 2024 and will be periodically evaluat
 
 If you had a legacy browser filter on before the update, the old filter will appear in your settings as "Deprecated". Deprecated legacy browser filters still work. However, if you turn them off, you won’t be able to turn them on again and will need to use the new filters instead.
 
+### Browser Extension Errors
+
+Some browser extensions are known to cause errors. You can use the browser extension filter to filter these out. The filter checks the error message and the event source to see if it's a known error coming from a browser extension.
+
+### Web Crawler Errors
+
+Due to their nature, web crawlers often encounter errors that normal users won't see. You can use the web crawler filter to filter out errors from web crawlers from the following sites: Baidu, Yahoo, Sogou, Facebook, Alexa, Slack, Google Indexing, Pingdom, Lytics, AWS Security Scanner, Hubspot, Bytedance, and other generic bots and spiders. Errors from Slack's Slackbot web crawler will not be filtered.
```
Comment (iker-barriocanal) [inline docs/product/data-management-settings/filtering/index.mdx:42]: Is it worth mentioning that we use the user-agent to check these?

**Review by iker-barriocanal — APPROVED** [review]

File: docs/product/data-management-settings/filtering/index.mdx (hunk 3 above)
Comment (roggenkemper) [reply docs/product/data-management-settings/filtering/index.mdx:42]: yeah, let me add something in!

**Pull request merged**